R/Simulations-class.R

```diff
@@ -409,6 +418,7 @@ DualSimulations <- function(rho_est,
       FinalTDEOTRatios = "numeric",
       FinalCIs = "list",
       FinalRatios = "numeric",
+      stop_report = "matrix",
```

Comment:
here it is ok though, because we just inherit from `GeneralSimulations` and that does not have this slot